assets/prefabs/recipe/arms/Gold/RecipeGoldAxe.prefab

```diff
@@ -0,0 +1,18 @@
+{
+    "ProcessDefinition": {
+        "processType": "EquipmentSmithing:ForgingTier3Process"
+    },
+    "CraftingStationRecipe": {
+        "recipeId": "EquipmentSmithing:GoldAxe",
+
+        "recipeComponents": ["3*Smithing:GoldBar"],
+        "recipeTools": ["1*forgeHammer"],
+        "requiredTemperature": 400,
```

Comment:
Temperature for forging gold seems to be around 500-600 according to [[2]].

[2]: http://www.siamcasting.com/casting_temperature_guide.html